### PR TITLE
feat: Add Discord social in the footer

### PR DESCRIPTION
**Pull request by marpit19**
## Related Issue 

- Information about the related issue 

Closes: # 

### Describe the changes you've made
* Add Discord social in the footer

* Add Discord social in the footer

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)
image

 Original           | Updated
 :--------------------: |:--------------------:
 original screenshot | updated screenshot |

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/web3community/.github/blob/main/CODE_OF_CONDUCT.md)

<a href="https://gitpod.io/#https://github.com/web3community/web3community.github.io/pull/138"></a>

### CODE REVIEW

File: src/_includes/partials/footer.html [hunk 1]
```diff
@@ -12,22 +12,26 @@
...
                         title='Connect with us at LinkedIn'></a>
 
                     <a aria-hidden='true'
-                        class='text-white fa fa-github fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
+                        class='text-white fab fa-github fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
                         href='https://github.com/web3community' rel='noreferrer' target='_blank' title='Contribute at GitHub'></a>
+                    
+                    <a aria-hidden='true'
+                        class='text-white fab fa-discord fa-lg transition duration-500 ease-in-out transform hover:-translate-y-1 hover:scale-100'
+                        href='https://discord.gg/TSRwqx4K2v' rel='noreferrer' target='_blank' title='Join our discord'></a>
```
Comment (Panquesito7) [inline src/_includes/partials/footer.html:34]:
```suggestion
                        href='https://discord.gg/TSRwqx4K2v' rel='noreferrer' target='_blank' title='Join our Discord community'></a>
```

**Review by Panquesito7 — COMMENTED** [review]
Would be nice if you could use the latest Discord logo. https://discord.com/branding

File: src/_includes/partials/footer.html (hunk 1 above)
Comment (marpit19) [reply src/_includes/partials/footer.html:34]: ok I'll change the logo and resolve this

**Comment by marpit19** [comment]
@Panquesito7 I made a new PR check this one out https://github.com/web3community/web3community.github.io/pull/139

**Pull request closed without merging**